src/Compilers/CSharp/Test/Emit2/FlowAnalysis/PatternMatchingTest5.cs

```diff
@@ -13,7 +13,9 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.UnitTests.FlowAnalysis
 {
+#pragma warning disable CA1052 // Static holder types should be Static or NotInheritable
     public class PatternMatchingTest5
+#pragma warning restore CA1052 // Static holder types should be Static or NotInheritable
```

Comment:
❓ What happens if this is marked `static`?

Reply:
I think it would work. I even think this class isn't needed. The only member in this class is `FlowAnalysisTests`, which shouldn't be a nested class IMO.

What do you think?

Reply:
Reverted everything under src/Compilers.